Formula/go.rb

```diff
@@ -91,8 +91,8 @@ def caveats
 
     if Hardware::CPU.arm?
       s += <<~EOS
-        This is a beta version of the Go compiler for Apple Silicon
-        (Go 1.16beta1).
+        This is a release candidate version of the Go compiler for Apple Silicon
+        (Go go1.16rc1).
```

Comment:
Should it just be `Go 1.16rc1` or `go1.16rc1`, and not a mix of both?

Reply:
Oh yeah, right 😅